src/blocks/stars.css

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-size: auto;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px 0 0;
```

Comment:
The `border-radius` property is set to `0.5px 0 0`, which might not achieve the desired effect. Typically, border-radius is used to round corners, and the values should be more consistent or intentional. Consider reviewing this property to ensure it meets your design requirements.